packages/camera/camera_avfoundation/lib/src/avfoundation_camera.dart

```diff
@@ -291,13 +300,17 @@ class AVFoundationCamera extends CameraPlatform {
   @override
   Stream<CameraImageData> onStreamedFrameAvailable(int cameraId,
       {CameraImageStreamOptions? options}) {
+    _installStreamController(onListen: _onFrameStreamListen);
+    return _frameStreamController!.stream;
+  }
+
+  void _installStreamController({Function()? onListen}) {
     _frameStreamController = StreamController<CameraImageData>(
-      onListen: _onFrameStreamListen,
+      onListen: onListen ?? () {},
```

Comment:
when do we want a no-op `onListen`

Reply:
Hopefully addressed in [this comment](https://github.com/flutter/plugins/pull/6290/#discussion_r957914180), but let me know if not.